run_tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+echo building js files
+npm run build
+echo build finished
+echo starting http server
+cd dist
+nohup  python3 -m http.server 8000 >/dev/null 2>&1 &
+serverPID=$!
+echo http server running PID $serverPID
+echo starting the tests
+cd ..
+export CYPRESS_baseUrl=http://localhost:8000
+npm run cypress:run
+echo tests finished
+echo killing http server
+kill $serverPID
+
```

Comment:
a jak by tam bol ten log tak tuto ho asi zmazat, lebo vsetko prebehlo v poriadku

Reply:
no ale je fakt ze ty v tom jednom kroku menis tu zlozku tak sa to bude musiet osetrit spravne, to mazanie logu

Reply:
tak jednoducho nebudeme log pridavat. Ani to neni v zadani. V zadani je vytvorit skript, ktory urobi build a pusti testy proti tomu buildu a nie proti dev prostrediu. Vysledky sa zobrazuju na obrazovku ako predtym. 

Log je nieco navyse, navrhujem vytvorit novy task, ak to tam chces.

Reply:
dobre ja som sa len tym snazil dostat nieco ked nastane nejaka necakana chyba tak aby sa dala interpretovat a nehladal to clovek hodinu